testing/regress/ecl-test-cluster160.json

```diff
@@ -57,12 +48,17 @@
             "soapcall_multihttpheader.ecl:TargetIP=."
         ],
         "engineParams":[
-            "failOnLeaks",
-            "disableLocalOptimizations"
```

Comment:
@AttilaVamos - I assume removed because no longer applicable/used?

Reply:
This is an example configuration file only. That parameter doesn't exists in the default ecl-test.json file and to keep consistency I removed it from here as well.

Reply:
But AFAIK they SHOULD be set on the default ecl-test.json and any time we run regression suite...

Reply:
Yes, it was a mistake to remove. Restored.